src/site/content/en/blog/user-agent-client-hints/index.md

```diff
@@ -0,0 +1,434 @@
+  returned, or populated with a varying value. This is referred to as
+  [GREASE](https://wicg.github.io/ua-client-hints/#grease).
+{% endAside %}
+
+### Example exchange
+
+An example exchange would look like this:
+
+⬆️ _Initial request from browser_<br> The browser is requesting the `/downloads`
+page from the site and sends its default basic User-Agent.
+
+```text
+GET /downloads HTTP/1.1
+Host: example.site
+
+Sec-CH-UA: "Chromium";v="84", "Google Chrome";v="84"
+Sec-CH-UA-Mobile: ?0
+```
+
+⬇️ _Response from server_<br> The server sends the page back and additionally
+asks for the full browser version and the platform.
+
+```text
+HTTP/1.1 200 OK
+Accept-CH: UA-Full-Version, UA-Platform
+
+<html>…
+```
+
+⬆️ _Subsequent requests_<br> The browser grants the server access to the
+additional information and sends the extra hints back in all subsequent
+responses.
+
+```text
+GET /downloads/app1 HTTP/1.1
+Host: example.site
+
+Sec-CH-UA: "Chromium";v="84", "Google Chrome";v="84"
+Sec-CH-UA-Mobile: ?0
+Sec-CH-UA-Full-Version: "84.0.4143.2"
+Sec-CH-UA-Platform: "Android"
+```
+
+### JavaScript API
+
+Alongside the headers, the User-Agent can also be accessed in JavaScript via
+`navigator.userAgentData`. The default `Sec-CH-UA` and `Sec-CH-UA-Mobile` header
+information can be accessed via the `brands` and `mobile` properties,
+respectively:
+
+```js
+console.log(navigator.userAgentData.brands);
+
+[
+  {
+    brand: 'Chromium',
+    version: '84',
+  },
+  {
+    brand: 'Google Chrome',
+    version: '84',
+  },
+];
+
+console.log(navigator.userAgentData.mobile);
+
+false;
+```
+
+The additional values are accessed via the `getHighEntropyValues()` call. The
+"high entropy" term is a reference to [information
+entropy](<https://en.wikipedia.org/wiki/Entropy_(information_theory)>), in other
+words - the amount of information that these values reveal about the user's
+browser. As with requesting the additional headers, it's down to the browser
+what values, if any, are returned.
+
+```js
+navigator
+  .userAgentData.getHighEntropyValues(
+    ["architecture", "model", "platform", "platformVersion",
+     "uaFullVersion"])
+  .then(ua => { console.log(ua) });
+
+{
+  "architecture": "x86",
+  "model": "",
+  "platform": "Linux",
+  "platformVersion": "",
+  "uaFullVersion": "84.0.4143.2"
+}
+```
+
+### Demo
+
+You can try out both the headers and the JavaScript API on your own device at
+[user-agent-client-hints.glitch.me](https://user-agent-client-hints.glitch.me).
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/user-agent-client-hints?path=README.md&previewSize=100"
+    title="user-agent-client-hints on Glitch"
+    allow="geolocation; microphone; camera; midi; vr; encrypted-media"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+{% Aside %}
+  The demo assumes that you're running a browser based on Chromium 84 or above.
+{% endAside %}
```

Comment:
It would be nice to also say this within the demo UI

Reply:
Will do. I need to update the demo readme and metadata too.